dist/apache.conf

```diff
@@ -0,0 +1,135 @@
+# This configuration file for Apache is mostly similar to the nginx example
+# configuration file.
+# Notable differences are:
+# - Assets aren't marked as immutable (but still have a 1y expire)
+# - No cache has been configured -- assuming that serving assets directly is
+#   good enough.
+# - no websocket is configured to the server on port 3000 -- I was assuming this
+#   isn't used, but maybe it's not true.
+# - The header X-Real-IP isn't added but X-Forwarded-For is, is it enough?
+#
+# Prerequisites:
+# Run: a2enmod expires proxy_http proxy_wstunnel rewrite headers deflate socache_shmcb
+# Copy paste the last part of Mozilla recommendation to configure TLS in your main config:
+# https://ssl-config.mozilla.org/#server=apache&version=2.4.41&config=intermediate&openssl=1.1.1d&guideline=5.4
+
+
+<VirtualHost *:80>
+	ServerAdmin email@example.org
+	ServerName  example.org
+
+	RedirectPermanent / https://example.org/
+
+	ErrorLog ${APACHE_LOG_DIR}/example.org-error.log
+
+	# Possible values include: debug, info, notice, warn, error, crit,
+	# alert, emerg.
+	LogLevel warn
+
+	CustomLog ${APACHE_LOG_DIR}/example.org-access.log combined
+</VirtualHost>
+
+
+<IfModule mod_ssl.c>
+<VirtualHost *:443>
+	ServerAdmin email@example.org
+	ServerName  example.org
+
+	ErrorLog ${APACHE_LOG_DIR}/example.org-ssl_error.log
+
+	# Possible values include: debug, info, notice, warn, error, crit,
+	# alert, emerg.
+	LogLevel warn
+
+  # Combined is defined like this in the main apache.conf file:
+  # LogFormat "%h %l %u %t \"%r\" %>s %O \"%{Referer}i\" \"%{User-Agent}i\"" combined
+
+	CustomLog ${APACHE_LOG_DIR}/example.org-ssl_access.log combined
+
+	DocumentRoot /home/mastodon/live/public
+	Header always set Strict-Transport-Security "max-age=63072000; includeSubDomains; preload"
+
+	# 80MB
+	LimitRequestBody 83886080
+
+	# LetsEncrypt
+  # You may have to configure this differently depending on your local setup
+	Alias "/.well-known/acme-challenge/" "/opt/mastodon/.well-known/acme-challenge/"
+	<Directory /opt/mastodon/.well-known/acme-challenge>
+		Require all granted	
+	</Directory>
+
+	# Additional configuration
+	<Directory /home/mastodon/live/public>
+		Require all granted
+	</Directory>
+
+	RewriteEngine On
+	# Serve local files if present, but otherwise proxy them
```

Comment:
Ideally it would have been nice to have a list of URL to redirect to the backend server instead of trying first locally, this is easier to use in Apache. But I decided to go the same path as the nginx configuration.